test/python/common/hdl.py

```diff
@@ -34,12 +33,21 @@ def install_stub(self, name):
                 setattr(stub, f, partial(h, timeout=self.timeout))
         return stub
 
+    def _readiness_check(self):
+        try:
```

Comment:
shouldn't we wait for a set amount of time until mayastor is ready? otherwise the tests will still fail if we're not immediately ready?

Reply:
Although it's not immediately clear but both bdev_list() and pool_list() actually invoke the appropriate gRPC call with wait_for_ready=True, which should queue the requests until the channel is ready. The problem is that there appears to be a bug in the python gRPC code where in some situations this doesn't always work as it is supposed to. It has been claimed that an immediate retry does seem to resolve the issue. That's what this change does, but it is difficult to verify as I have been unable to reproduce the conditions under which the bug is triggered. At the very least, the change should not make matters worse.

Reply:
Will update the commit message to be more informative